CHANGELOG.md

```diff
@@ -6,10 +6,16 @@ ENHANCEMENTS:
 * `d/tfe_workspace`: Add an `auto_destroy_at` attribute for reading a scheduled auto-destroy, by @notchairmk [1354](https://github.com/hashicorp/terraform-provider-tfe/pull/1354)
 * `r/tfe_registry_module`: Add `initial_version` support for Branch Based Modules by @aaabdelgany [#1363](https://github.com/hashicorp/terraform-provider-tfe/pull/1363)
 * `r/tfe_oauth_client`: Add `agent_pool_id` as an optional argument to enable Private VCS support, by @roleesinhaHC [1255](https://github.com/hashicorp/terraform-provider-tfe/pull/1255)
+* `d/tfe_organization_run_task_global_settings`: Added a datasource to retrieve the global settings of Run tasks, by @glennsarti [#1328](https://github.com/hashicorp/terraform-provider-tfe/pull/1330)
```

Comment:
nit: are these new data source/resources? Should it be under features instead of enhancement. Also just for consistency with the other changelog entries, we could active voice instead of passive (Adds instead of Added).

Reply:
Done